net/http/src/THttpWSHandler.cxx

```diff
@@ -84,7 +84,8 @@ THttpWSHandler::~THttpWSHandler()
 
    {
       std::lock_guard<std::mutex> grd(fMutex);
-      clr = std::move(fEngines);
+      for (auto &eng : fEngines)
+         clr.emplace_back(std::move(eng));
```

Comment:
I would have expected these two to behave the same for your case (move storage vs move each element). Could you add a comment why we cannot move the storage?

Reply:
Of course, I was not aware (forgot) about `std::swap()` function